.github/workflows/operator-test.yaml

```diff
@@ -58,7 +58,8 @@ jobs:
         run: |
           helm install --namespace=opentelemetry-operator-system --create-namespace my-opentelemetry-operator ./charts/opentelemetry-operator \
             --set 'manager.extraArgs[0]="--enable-go-instrumentation"' \
-            --set "manager.collectorImage.repository=otel/opentelemetry-collector-k8s"
+            --set "manager.collectorImage.repository=otel/opentelemetry-collector-k8s" \
+            --set "manager.podLabels.control-plane=controller-manager"
```

Comment:
OOC why do we need these labels now?

Reply:
[The tests were failing ](https://github.com/open-telemetry/opentelemetry-helm-charts/actions/runs/12193933388/job/34016968409#step:13:116) due to https://github.com/open-telemetry/opentelemetry-operator/pull/3486 which let to `e2e/operator-restart` delete step not matching the operator pod.
The alternative was to add the `Values.manager.podLabels.control-plane` label, but I don't think it makes sense to have this label in the default values.